projects/lib/src/board/board.cpp

```diff
@@ -131,12 +131,15 @@ int Board::maxPieceSymbolLength() const
 void Board::setPieceType(int type,
 			 const QString& name,
 			 const QString& symbol,
-			 unsigned movement)
+			 unsigned movement,
+			 const QString& gSymbol)
 {
 	if (type >= m_pieceData.size())
 		m_pieceData.resize(type + 1);
 
-	PieceData data = { name, symbol.toUpper(), movement };
+	const QString & gsym = gSymbol.isEmpty() ? symbol : gSymbol;
```

Comment:
In our convention the `&` character comes immediately after the type without a space between them.